### ISSUE TITLE
The silnet's output mask is not a binary mask. but more like a detected edge!!!

### ISSUE THREAD

**Issue by visonpon** [Ninebell/GaneratedHandsForReal_TIME#14]

**Comment by Ninebell** [comment]
sorry, sir what did you mean??
Did you mean silnet output is image of edge, not binary mask?

**Comment by visonpon** [comment]
@Ninebell yeah, I tested the trained model you provided on my own image, and get the strange result, hope you can give some advice, thanks~

**Comment by Ninebell** [comment]
@visonpon 
can you show me some examples?

**Comment by Ninebell** [comment]
@visonpon 
trained silnet model can work image which background is white

**Comment by visonpon** [comment]
![1938378466](https://user-images.githubusercontent.com/11517731/71068646-1c3ee280-21b2-11ea-9ecb-e4f0e5848d51.jpg)
oh, my test images' background is green, maybe this is why I got bad results.
btw, I have trained silnet on my own data, but at the beginning of the training phase, the accuracy has achieved to 80+, I also curious about this...

**Comment by Ninebell** [comment]
@visonpon 
Is dataset personal?? or Just your dataset?
